app/main/NameTag.tsx

```diff
@@ -101,11 +101,14 @@ function NameTag({
       <div>
         <h2 style={{ fontSize: '30px', fontWeight: 'bold', display: 'inline-block' }}>Name Tag</h2>
         <ColoredSwitch
-          role='switch'
-          data-testid = 'colored-switch'
-          aria-checked={showNametag}
-          checked={showNametag}
-          onChange={() => setShowNametag(!showNametag)}
+          inputProps={{
+            role: 'switch',
```

Comment:
According to https://mui.com/material-ui/react-switch/#accessibility, we should test that the assistive technology our users will use supports the `switch` role properly before setting this. Have we done that?